src/app.js

```diff
@@ -1 +1,19 @@
-// write code here
+const fs = require('fs');
+const path = require('path');
+
+(() => {
+  try {
+    const [source, dest] = process.argv.slice(2);
```

Comment:
Consider adding a check to ensure that both `source` and `dest` are provided as command-line arguments. If they are not, you could log an error message and exit the process early to prevent runtime errors.